jujugui/static/gui/src/app/components/accordion-section/accordion-section.js

```diff
@@ -85,12 +79,11 @@ class AccordionSection extends React.Component {
     @return {Object} The React div element.
   */
   _generateContent() {
-    return (<div
-      className="accordion-section__content"
-      ref="content"
-      style={this._getStyle()}>
-      {this.props.children}
-    </div>);
+    return (
+      <div className="accordion-section__content" ref="content" style={this._getStyle()}>
```

Comment:
Personal preference, but I much prefer the 
```
(
  </>
);
```

format to
```
(<
/>)
```

The indentation points to the important part of the return